bin/etl.js

```diff
@@ -1,130 +1,120 @@
 #!/usr/bin/env node
-'use strict';
-const { Client } = require('pg');
-const copyFrom = require('pg-copy-streams').from;
-const http = require('http');
-const path = require("path");
+/* eslint-disable no-console */
+
 const fs = require('fs');
+const request = require('request');
 const unzip = require('unzip');
-const copyTo = require('pg-copy-streams').to;
-const { exec } = require('child_process');
 const iconv = require('iconv-lite');
+const copyFrom = require('pg-copy-streams').from;
+const { Client } = require('pg');
 
+const downloadFile = (url, fileName) =>
+  new Promise((resolve, reject) => {
+    console.log('Downloading file');
+    request(url)
+      .on('error', (err) => {
+        console.log(err);
+        reject(err);
+      })
+        .pipe(fs.createWriteStream(fileName))
+        .on('close', () => {
+          resolve('success');
+        });
+  });
 
-// Ensure environment variables are read.
-require('../config/env');
+const unzipFile = (pathToZip, pathToExtract) =>
+ new Promise((resolve, reject) => {
+   console.log('Extracting file from zip');
+   fs.createReadStream(pathToZip)
+      .pipe(unzip.Extract({ path: pathToExtract }))
+      .on('error', (error) => {
+        console.log(error);
+        reject(error);
+      })
+      .on('close', () => {
+        resolve();
+      });
+ });
 
-if (process.env.NODE_ENV !== 'production') {
-  let dotenv = require('dotenv');
-  dotenv.config();
-}
+const truncateTable = async () => {
+  console.log('Truncating table');
+  return await client.query(`TRUNCATE TABLE ${process.env.DB_TABLE};`);
+};
 
-const the_url = 'http://dl.ncsbe.gov/data/ncvoter41.zip';
-const zip_filename = `${__dirname}/tmp/ncvoter41.zip`;
-const expected_filename = 'ncvoter41.txt';
-const extracted_filename = `${__dirname}/tmp/${expected_filename}`;
-const extracted_filename_utf8 = `${extracted_filename}.utf8`;
+const deleteResidentAddressField = async () => {
+  console.log('Removing resident_address column');
+  return await client.query(`ALTER TABLE ${process.env.DB_TABLE} DROP COLUMN IF EXISTS resident_address`);
```

Comment:
Same thing here. No need to `return await` since the `await` on the function invocation will wait for this promise to resolve.